packages/cozy-procedures/src/components/Overview.jsx

```diff
@@ -45,6 +45,26 @@ class Overview extends React.Component {
     if (job.state === 'done') {
       Alerter.success(t('overview.zip_ready'))
       client.collection('io.cozy.files').destroy(jsonFile)
+    } else if (job.state === 'errored') {
+      Alerter.error(t('overview.zip_errored'))
+      // if an error occured, destroy the zip and the json
+      client
+        .collection('io.cozy.files')
+        .find(
+          {
+            dir_id: job.message.dir_id,
+            name: job.message.filename
+          },
+          {
+            indexedFields: ['dir_id', 'name']
+          }
+        )
+        .then(resp => {
```

Comment:
why not an await ?

Reply:
as it is passed as a callback to `realtime.subscribe` I can not `await` it. When I first implemented it I had issues with it but in fact we don't care, I can still give an async callback. I changed it.